README.md

```diff
@@ -1 +1,60 @@
-# Hello_Microverse
\ No newline at end of file
+Hello_Microverse
+
+_______________________________________________________________________________
+
+
+Additional description about the project and its features.
+
+Built With
+Major languages
+Frameworks
+Technologies used
+
+_______________________________________________________________________________
+Live Demo
+Live Demo Link
+_______________________________________________________________________________
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+
+Setup
+
+Install
+
+Usage
+
+Deployment
```

Comment:
- [ ] If you do not need these sections, please delete them to keep `readme` clean 😉

Reply:
ReadMe has been updated.